packages/validation/src/loadbalancers.schema.ts

```diff
@@ -155,36 +155,42 @@ export const UpdateRouteSchema = object({
 });
 
 export const UpdateConfigurationSchema = object({
-  label: string().min(1),
-  port: number(),
+  label: string().min(1, 'Label must not be empty.'),
+  port: number()
+    .min(0, 'Port must be greater than 0.')
+    .max(65_535, 'Port must be less than 65535.')
```

Comment:
This validation works... but was the `_` intentional or a typo?

```suggestion
    .max(65_535, 'Port must be less than 65535.')
```